.gitignore

```diff
@@ -20,8 +20,8 @@ __pycache__/
 *.so.dSYM
 *.dll
 #Allow
-!qiskit_nature/drivers/gaussiand/gauopen/*.so
-!qiskit_nature/drivers/gaussiand/gauopen/*.pyd
+!qiskit_nature/drivers/second_quantization/gaussiand/gauopen/*.so
+!qiskit_nature/drivers/second_quantization/gaussiand/gauopen/*.pyd
```

Comment:
Since the drivers have been kept in their old place for now too, we should have both of the sections in there right, (ie do not delete the original lines 23/24)

Reply:
Thanks for spotting this! Reverting the moving was a rather manual process in some cases because I did not want to revert the entire commit. I missed this aspect. Will fix asap